virttest/test_setup/libvirt.py

```diff
@@ -0,0 +1,27 @@
+from virttest.test_setup.core import Setuper
```

Comment:
we already have one file named libvirt.py for other purpose under virttest/utils_test/libvirt.py .
Can we have other name e.g libvirt_setup.py to avoid confusion